extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -110,14 +98,29 @@ define('ext.wikia.adEngine.slotTweaker', [
 		}
 	}
 
+	function getRecoveredIframe(slotName) {
+		var fallbackId = win._sp_.getElementId(document.querySelectorAll('div[id="' + slotName + '"] div')[1].id);
+		return doc.getElementById(fallbackId).querySelector('div:not(.hidden) > div[id*="_container_"] iframe');
```

Comment:
same here, maybe it's worth checking if element exists before calling `querySelector` on it.